recipe-server/client/control/components/common/NavigationMenu.js

```diff
@@ -56,6 +73,8 @@ export default class NavigationMenu extends React.PureComponent {
                 </Item>),
               )
             }
+
+            {importExperiments}
```

Comment:
This should be moved to the top of the menu - currently it is lost in the 'most recently viewed' list.